src/core/99_variables.tf

```diff
@@ -306,6 +306,11 @@ variable "cidr_subnet_fnlollipop" {
   description = "Function Lollipop address space."
```

Comment:
Because the application gateway is in the `core` space. Moreover the dynamic link QR code is a general pourpose functionality, not binded to any domain/product

Reply:
is it possibile to isolate all the changes into a new directory ie. `dynamic-link`?

Reply:
It is possible to isolate only the AppService (and related variables) in a separate directory (which will require setting up a separate pipeline for it).

However the AppGateway integration part has to remain in the `core` directory